app/build.gradle.kts

```diff
@@ -111,6 +111,14 @@ android {
             setDimension(ProjectSettings.Flavor.DIMENSION)
         }
     }
+
+    lintOptions {
+        textReport = true             // Write a text report to the console (Useful for CI logs)
+        isExplainIssues = false       // HTML/XML reports are too verbose in console logs
+        isCheckDependencies = false   // Required to get all unused resource from other modules (disabled to speed up linting)
+        isCheckTestSources = true     // Also check test case code for lint issues
+        isCheckReleaseBuilds = false  // If we run a full lint analysis as build part in CI, we can skip redundant checks
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/55fa8bb336f7dbb40705d8c6244a3a3c17d0f9e1/app/build.gradle.kts#L120"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>